fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/rule/AggregateJoinPushDownRule.java

```diff
@@ -79,9 +78,6 @@ public boolean check(OptExpression input, OptimizerContext context) {
         if (Utils.isOptHasAppliedRule(input, Operator.OP_PUSH_DOWN_BIT)) {
             return false;
         }
-        if (!MvUtils.isLogicalSPJG(input)) {
```

Comment:
Removing this condition is too open that would be risky,  can we use a conservative condition, for an example:
```java
MvUtils.onlyContainsSPGJ(input);
```

onlyContainsSPGJ can support sub-plans as follows:
1. aggregation over single-table, this scenarios are your expect;
2. aggregation over multi-join, this just what MvUtils.isLogicalSPJG() means;